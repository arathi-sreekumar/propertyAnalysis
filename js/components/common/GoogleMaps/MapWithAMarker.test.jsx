import React from 'react';
import { shallow } from 'enzyme';
import GoogleMapWithMarker from './MapWithAMarker';

const MAP_CENTER = {
	lat: 50.7983845,
	lng: 0.001313
};
const ZOOM = 11;
const BOUNDS = { sw: { lat: 50.785809, lng: -0.025241 }, ne: { lat: 50.81096, lng: 0.027867 } };

describe('GoogleMapWithMarker', () => {
	it('renders correctly', () => {
		const component = shallow(<GoogleMapWithMarker center={MAP_CENTER} zoom={ZOOM} bounds={BOUNDS} />);
		expect(component).toMatchSnapshot();
	});
});
