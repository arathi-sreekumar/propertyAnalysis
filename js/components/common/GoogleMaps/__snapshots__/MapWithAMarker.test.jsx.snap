// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleMapWithMarker renders correctly 1`] = `
<withScriptjs(withGoogleMap(Component))
  bounds={
    Object {
      "ne": Object {
        "lat": 50.81096,
        "lng": 0.027867,
      },
      "sw": Object {
        "lat": 50.785809,
        "lng": -0.025241,
      },
    }
  }
  center={
    Object {
      "lat": 50.7983845,
      "lng": 0.001313,
    }
  }
  containerElement={
    <div
      style={
        Object {
          "height": "400px",
        }
      }
    />
  }
  googleMapURL="https://maps.googleapis.com/maps/api/js?key=AIzaSyCBUAx3W_ykWX9gSePohkLo-TRTeNV9iwI&v=3.exp&libraries=geometry,drawing,places"
  loadingElement={
    <div
      style={
        Object {
          "height": "100%",
        }
      }
    />
  }
  mapElement={
    <div
      style={
        Object {
          "height": "100%",
        }
      }
    />
  }
  zoom={11}
/>
`;
